src/dafny/Relations.dfy

```diff
@@ -52,4 +52,13 @@ module {:options "-functionSyntax:4"} Dafny.Relations {
     && Connected(R)
   }
 
+  /** An element in an ordered set is called minimal, if it is less than every other element of the set. */
+  ghost predicate IsMinimum<T>(R: (T, T) -> bool, m: T, s: set<T>) {
+    m in s && forall y | y in s :: R(m, y)
```

Comment:
The variable name `m` and `y` aren't very descriptive. I would use `element` and `other`, respectively.